### PR TITLE
chore: adjust raw `near call` to newer `near js` calls in README

### PR DESCRIPTION
**Pull request by ChaoticTempest**
Converted over most of the `near call ...` into `near js ...` ones. I also corrected some IDs that didn't look right, so let me know if those make sense.

Also, hid the output in html tags so that the README isn't too long and cognitive load becomes a problem when a new user comes to read it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,109 +75,215 @@ target/debug/neard run
...
-Starting deployment. Account id: test.near, node: http://localhost:3030, helper: http://localhost:3000, file: jsvm.wasm
-Transaction Id EGVd29tthMp7fqkDgP8frftgZhhb3FVazaFhvXpYXNhw
-To see the transaction in the transaction explorer, please open this url in your browser
-http://localhost:9001/transactions/EGVd29tthMp7fqkDgP8frftgZhhb3FVazaFhvXpYXNhw
-Done deploying to test.near
+near-sdk-js (master) near deploy test.near jsvm.wasm
 ```
 
+<details>
+<summary><strong>Example output</strong></summary>
```
Comment (ailisp) [inline README.md:84]: Nice I like the example output! Much cleaner than before

File: examples/cross-contract-call/README.md [hunk 2]
```diff
@@ -21,26 +21,54 @@ yarn test
...
 ## Call the contract
 Under the root dir of near-sdk-js, call the `set_status` and `set_person_on_call` using this pattern:
 
+```sh
+near js call <contract-id> <function-name> [--args '<parameter>'] --deposit 0.1 --accountId <signer-id> --jsvm jsvm.test.near
 ```
-near call jsvm.test.near call_js_contract --accountId <accountID> --base64 --args $(node encode_call.js <contract-account-id> <function-name> '[<parameter>]') --deposit 0.1
-```
+
+or with t<details>
```
Comment (ailisp) [inline examples/cross-contract-call/README.md:67]: typo here?

Comment (ailisp) [reply examples/cross-contract-call/README.md:67]: Also suggest to drop the raw commands in these example READMEs. I think just let advanced user know about the raw command in one place?

**Review by ailisp — APPROVED** [review]
Thanks, it's a comprehensive update! Except the typo, the other comments are just my opinions, feel free to merge it if you think it's ready.

**Review by volovyk-s — APPROVED** [review]
Awesome! We will need to reorg README in the future. Move low-level APIs to a separate file, add simple contract examples, reorg examples READMEs, etc. But no blockers here!

**Pull request merged**